### PR TITLE
Sort commands

### PR DESCRIPTION
**Pull request by gabriel**
The help commands we're sorted, so am sorting them.

### CODE REVIEW

**Comment by gabriel** [comment]
cc @maxtaco

**Comment by maxtaco** [comment]
Looks good, why the updated CLI vendoring though?

File: go/vendor/github.com/keybase/cli/command.go [hunk 1]
```diff
@@ -201,3 +201,17 @@ func (c Command) startApp(ctx *Context) error {
...
+func (c ByName) Len() int {
+	return len(c)
+}
+
+func (c ByName) Swap(i, j int) {
+	c[i], c[j] = c[j], c[i]
+}
+
+func (c ByName) Less(i, j int) bool {
+	return strings.Compare(c[i].Name, c[j].Name) == -1
```
Comment (patrickxb) [inline go/vendor/github.com/keybase/cli/command.go]: No need for strings.Compare:

    return c[i].Name < c[j].Name

Comment (gabriel) [inline go/vendor/github.com/keybase/cli/command.go]: Fixed thx

**Comment by gabriel** [comment]
I put the sort function in cli package so needed to update the vendor with that change

**Pull request merged**